### PR TITLE
Qwigley: Version 1.010; ttfautohint (v1.8.3) added

### PR DESCRIPTION
**Pull request by vv-monsalve**
43b48aa: [gftools-packager] Qwigley: Version 1.010; ttfautohint (v1.8.3) added

* Qwigley Version 1.010; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/googlefonts/qwigley at commit https://github.com/googlefonts/qwigley/commit/cef3747a8662bd9ec4172a8c1b6866051ac47d6c.

 9d1650b: [gftools-packager] ofl/qwigley remove METADATA "source".  google/fonts#2587

### CODE REVIEW

**Comment by vv-monsalve** [comment]
Updated

## Qwigley: Version 1.010; ttfautohint (v1.8.3) added

---
 ba562bf: [gftools-packager] Qwigley: Version 1.010; ttfautohint (v1.8.3) added

* Qwigley Version 1.010; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/googlefonts/qwigley at commit https://github.com/googlefonts/qwigley/commit/7a963366b5efe19ef39603d03888a2105105dc01.

 21a5de8: [gftools-packager] ofl/qwigley remove METADATA "source".  google/fonts#2587

**Comment by vv-monsalve** [comment]
Update done changing to a single empty line.

**Comment by m4rc1e** [comment]
Thanks. LGTM.

File: ofl/qwigley/OFL.txt [hunk 1]
```diff
@@ -1,94 +1,92 @@
...
-INCLUDING ANY GENERAL, SPECIAL, INDIRECT, INCIDENTAL, OR CONSEQUENTIAL
-DAMAGES, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-FROM, OUT OF THE USE OR INABILITY TO USE THE FONT SOFTWARE OR FROM
-OTHER DEALINGS IN THE FONT SOFTWARE.
+Copyright 2011 The Qwigley Project Authors (https://github.com/googlefonts/qwigley)
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
+This license is copied below, and is also available with a FAQ at:
+https://scripts.sil.org/OFL
+
```
Comment (m4rc1e) [inline ofl/qwigley/OFL.txt:6]: We should definitely automate the generation of these OFL.txt files. If we compare against the previous, we're missing another line break.

Comment (m4rc1e) [reply ofl/qwigley/OFL.txt:6]: I've just added the break myself.

**Pull request merged**